packages/core/src/utils/configureMount.ts

```diff
@@ -48,6 +49,10 @@ export const configureMount: ConfigureMount = ({
         },
       });
 
+      if (appProps && !isObject(appProps)) {
+        throw new Error(`onBeforeMount must retun an object or void`);
```

Comment:
V. Minor - `return`

Reply:
What do you mean by `return`? if it throws the return won't be executed, will it?

Reply:
haha sorry, I literally meant there was a typo` `retun` -> `return`

Reply:
I see, I'm so good a typos that I don't see them even when someone points them out 😄